sklearn/cluster/affinity_propagation_.py

```diff
@@ -197,7 +197,10 @@ class AffinityPropagation(BaseEstimator, ClusterMixin):
     Parameters
     ----------
     damping : float, optional, default: 0.5
-        Damping factor between 0.5 and 1.
+        Damping factor (between 0.5 and 1) damps the
```

Comment:
How about "Damping factor (between 0.5 and 1) is the extent to which the current value is maintained relative to incoming values (weighted `1 - damping`)."?